client/screens/Hooks/useFeedLogic.tsx

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from 'expo-router';
```

Comment:
these hooks should all be in the /hooks folder, in their own subfolders for organization. currently it is in /screens/hooks